internal/resources/datasource_available_resources.go

```diff
@@ -216,6 +217,14 @@ func volumeCollectionResource() *schema.Resource {
 				Computed:    true,
 				Description: "Description of the volume collection",
 			},
+			vcStoragePoolIDs: {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Description: "A list of valid storage pool ids.",
```

Comment:
minor, but the other descriptions do not end in a period.  Also, could remove "valid" since it seems that is implied.